cmd/manager/main.go

```diff
@@ -171,6 +175,19 @@ func main() {
 	}
 }
 
+func printConfig(cfg *configuration.Config) {
+	logWithValuesMemberOperator := log
+	for key, value := range cfg.GetAllMemberParameters() {
+		logWithValuesMemberOperator = logWithValuesMemberOperator.WithValues("key", key, "value", value)
```

Comment:
Actually, `getMemberEnvVarKey` was copied by accident. I meant to log the values in the same way that the registration config is logged here: https://github.com/codeready-toolchain/host-operator/blob/d764a1c431ef2ad964ca374dbb34100f8d094c0b/cmd/manager/main.go#L233

So I will just remove the `getMemberEnvVarKey` function since it's not needed.

Reply:
Done as part of ddbbefc0a4adaa34d70562e367889177f2cb83a3